src/grandorgue/GOOrganController.cpp

```diff
@@ -219,22 +219,24 @@ void GOOrganController::ReadOrganFile(GOConfigReader &cfg) {
     else
       m_InfoFilename = wxEmptyString;
   } else {
-    GOLoaderFilename fname;
-
-    fname.Assign(info_filename);
-    std::unique_ptr<GOOpenedFile> file = fname.Open(m_FileStore);
-    fn = info_filename;
-    if (
-      file->isValid()
-      && (fn.GetExt() == wxT("html") || fn.GetExt() == wxT("htm"))) {
-      if (fn.FileExists() && !m_FileStore.AreArchivesUsed())
+    if (!m_FileStore.AreArchivesUsed()) {
+      info_filename.Replace(
+        wxT("\\"), wxString(wxFileName::GetPathSeparator()));
+      wxString infoFilePath
+        = wxFileName(GetODFFilename()).GetPath(wxPATH_GET_SEPARATOR)
+        + info_filename;
```

Comment:
This code duplicates one in `GOLoaderFilename::Open`. Please extract it into a new method of `GOLoaderFilename`

Reply:
But this method may be a static one. It would not require creating an object.

Reply:
Yes, that might work. I'll test it and see.